.gitignore

```diff
@@ -313,6 +322,143 @@ Temporary Items
 # iCloud generated files
 *.icloud
 
+### Node ###
```

Comment:
No particular harm, but is there any reason you added in Node-related ignores?
Edit: Nevermind, found it!